utilities/OnYourDataIngestionAPI.ps1

```diff
@@ -2,71 +2,71 @@
 
 # Azure Open AI configuration
 
-$AzureOpenAIEndpoint = "<resource-name>.openai.azure.com"
-$EmbeddingDeploymentName = "embedding"
+$AzureOpenAIEndpoint = "knop-swedencentral-azopenai.openai.azure.com"
```

Comment:
I'd revert this back to how you had it originally without specific endpoint names.